app/javascript/mastodon/features/ui/components/media_modal.js

```diff
@@ -104,21 +134,40 @@ export default class MediaModal extends ImmutablePureComponent {
       alignItems: 'center', // center vertically
     };
 
+    const navigationClassName = classNames('media-modal__navigation', {
+      'media-modal__navigation--hidden': !navigationShown,
```

Comment:
If CSS style uses `media-modal__navigation--hidden`, why don't you keep the state as `navigationHidden`?